Packages/com.unity.inputsystem/InputSystem/Devices/Gamepad.cs

```diff
@@ -255,31 +255,31 @@ public enum GamepadButton
         /// The upper action button on a gamepad.
         /// </summary>
         /// <remarks>
-        /// Identical to <see cref="Y"/> and <see cref="Triangle"/> which are the Xbox and PlayStation controller names for this button.
+        /// Identical to <see cref="North"/> and <see cref="North"/> which are the Xbox and PlayStation controller names for this button.
```

Comment:
This seems incorrect? Referencing to the same thing

Reply:
I believe this is due to my first pass where I removed the aliased values. Will revert